### PR TITLE
Update shopware installation process on MacOS

### PR DESCRIPTION
**Pull request by sushmangupta**

### CODE REVIEW

File: 1.Familiarize yourself with Shopware/2.Setting Up Your Shopware Store/1.Installation process - Local installation vs. hosted solutions.md [hunk 1]
```diff
@@ -1,5 +1,187 @@
...
+
+Open your terminal and run the following command to install Nix:
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install
+```
+
+![nix](./LP1-CO2-LU1/nix.jpg)
+
+### Installation of Zsh (Optional)
```
Comment (dennishoeting) [inline 1.Familiarize yourself with Shopware/2.Setting Up Your Shopware Store/1.Installation process - Local installation vs. hosted solutions.md:50]: should this not be right after installing iTerm2?

File: 1.Familiarize yourself with Shopware/2.Setting Up Your Shopware Store/1.Installation process - Local installation vs. hosted solutions.md [hunk 2]
```diff
@@ -1,5 +1,187 @@
...
+
+Zsh is an optional tool to enhance your console's appearance and usability. If you wish to install it, explore further on your own.
```
Comment (dennishoeting) [inline 1.Familiarize yourself with Shopware/2.Setting Up Your Shopware Store/1.Installation process - Local installation vs. hosted solutions.md:52]: can we provide a link here?

**Review by dennishoeting — APPROVED** [review]
Approved, let's get it in :) 

My only comment would be that in the steps, we could elaborate on putting things in context - what is the idea behind nix, cachix, devenv, composer .. however, at 2nd thought I think this is something that CAN be in the video but does not have to be in the text nessesarily.. OR we find a way to get over side notes...
Overall, I like how this unit is so straight forward! 
Let's get it into the hub 👍

File: 1.Familiarize yourself with Shopware/2.Setting Up Your Shopware Store/1.Installation process - Local installation vs. hosted solutions.md (hunk 1 above)
Comment (sushmangupta) [reply 1.Familiarize yourself with Shopware/2.Setting Up Your Shopware Store/1.Installation process - Local installation vs. hosted solutions.md:50]: yes. Infact we can change the order

File: 1.Familiarize yourself with Shopware/2.Setting Up Your Shopware Store/1.Installation process - Local installation vs. hosted solutions.md (hunk 2 above)
Comment (sushmangupta) [reply 1.Familiarize yourself with Shopware/2.Setting Up Your Shopware Store/1.Installation process - Local installation vs. hosted solutions.md:52]: Also at the moment we can skip this. This is an additional step that does not relate to installation. So I have remove it. Later we can also add the link of necessary : https://fmacedoo.medium.com/oh-my-zsh-with-powerline-fonts-pretty-simple-as-you-deserve-fbe7f6d23723

**Pull request merged**